gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -244,49 +243,73 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
     return;
   }
 
-  // Load the RobotHWSim abstraction to interface the controllers with the gazebo model
+  std::unique_ptr<hardware_interface::ResourceManager> resourceManager_ =
+    std::make_unique<hardware_interface::ResourceManager>();
```

Comment:
I deliberately don't highlight changes in style, but you're using camelCase here, whereas other variables use snake_case. Just fyi.